AppCenter/AppCenter/Internals/Util/MSUtility+File.m

```diff
@@ -166,21 +170,24 @@ + (NSURL *)appCenterDirectoryURL {
 #else
     dirURL = [baseDirUrl URLByAppendingPathComponent:kMSAppCenterBundleIdentifier];
 #endif
-    if (![dirURL checkResourceIsReachableAndReturnError:nil]) {
-      [self createDirectoryAtURL:dirURL];
-    }
+    [self createDirectoryAtURL:dirURL];
```

Comment:
Should we check the return value and log any errors here?

Reply:
About logging errors: We are logging the error inside `createDirectoryAtURL:`, do you think we need more logging there?

Reply:
Oh.. I see. Then probably we don't need it.